flow/project.py

```diff
@@ -1663,11 +2051,11 @@ def _fetch_scheduler_status(self, jobs=None, file=None, ignore_errors=False):
             scheduler_info = {sjob.name(): sjob.status() for sjob in self.scheduler_jobs(scheduler)}
             status = dict()
             print("Query scheduler...", file=file)
-            for job in tqdm(jobs,
-                            desc="Fetching operation status",
-                            total=len(jobs), file=file):
-                for group in self._groups.values():
-                    _id = group._generate_id(job)
+            for group in self._groups.values():
+                _ids = group._get_filtered_job_id_per_group(jobs)
```

Comment:
What happens if a subset of jobs is passed and then the aggregates created are not the ones used by default?

Reply:
This concept can get confusing because there are many possibilities for this one. 
If a user does as you said, then the user should expect that the aggregates that will be formed won't be the same as expected.
I will soon come up with an efficient way of dealing with this.